README.md

```diff
@@ -160,6 +160,42 @@ grunt.initConfig({
 });
 ```
 
+#### <a id="optionsonly">options.only</a>
+ * type: `string` or `array of strings`
+ * default: `'.'`
+
+When options.add is false, you may specify a filter to select the files to remove, instead of removing all files.
+
+Example of the `only` option:
+
+```js
+/**
+ * The task below will only remove the index.html and .js files from the `gh-pages` branch before copying over files from the `src`.
```

Comment:
This comment would be easier to read if it were hard wrapped (around 80 chars is good).  See the hosted version of [your readme](https://github.com/BillyNate/grunt-gh-pages/blob/master/README.md).  It would be nice not to have to scroll to read the whole line.